system/jlib/jptree.cpp

```diff
@@ -6863,9 +6868,8 @@ class CPullJSONReader : public CJSONReaderBase<X>, implements IPullPTreeReader
             expecting(",");
         return true;
     }
-    void newAttribute()
+    void newNamedAttribute()
     {
-        readName(tag.clear());
         skipWS();
```

Comment:
@jakesmith I think for the following JSON:

"@myattr":  "value",

the prior call (outside this function) to readName would leave the position at the ':'.  In which case this skipWS would move the position to the beginning of "value"